Assets/MixedRealityToolkit-SDK/Features/Diagnostics/DiagnosticBehavor.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace Assets.MixedRealityToolkit_SDK.Features.Diagnostics
+{
+    public class DiagnosticBehavior : MonoBehaviour
```

Comment:
shouldn't this have a `IMixedRealityDiagnosticsHandler`?

Reply:
Should it? I assumed since it's an internal behavior that the manager driving it was sufficient. Maybe it makes sense to hook up with the interface though.

Reply:
Well I noticed you're just setting it outright. bc you have a reference to it. 

But that would also remove the need to keep a cached reference to this component after you add it to your Quad

Reply:
Got it, that seems much cleaner.